appraisal.lic

```diff
@@ -95,6 +113,38 @@ class Appraisal
     waitrt?
     bput('wear my zill', 'You slide')
   end
+
+  def appraise_focus(item)
+    case bput("get my #{item}", "^You get.*#{item}", '^What were you referring', 'But that is already', 'You are already')
+    when 'But that is already', '^What were you referring'
+      echo "Cannot get the item you are trying to focus upon. Exiting..."
```

Comment:
💡 Suggestion: include `item` in the echo, something like `"Cannot get item '#{item}'.....`